### ISSUE TITLE
Adding delay in the start of malicious mote.

### ISSUE THREAD

**Issue by abhiverma866** [dhondta/rpl-attacks#41]
I want to add delay in the node startup of malicious mote. My aim is to make a mote send its first DIS after certain period of time. Like in case of hello-flood the DIS_START_DELAY is set to 0. However, when i set delay to 60, it rarely makes any difference. The hello-flood should start after the nodes have found their parents i.e. network topology has been setup. How can i do this. 
![screenshot from 2019-01-30 12-27-44](https://user-images.githubusercontent.com/16736201/51963831-c4f52e00-248a-11e9-8626-0b7dbf5e1f0f.png)
![screenshot from 2019-01-30 12-28-10](https://user-images.githubusercontent.com/16736201/51963833-c58dc480-248a-11e9-8157-e62ed4d80bb7.png)
![screenshot from 2019-01-30 12-28-43](https://user-images.githubusercontent.com/16736201/51963836-c58dc480-248a-11e9-9d14-c8384eafe6ba.png)

**Comment by dhondta** [comment]
Hi @abhiverma866 
I apologize for the huge delay.

After a bit of testing, it appears that, whatever I set for the related constant, in practice the delay always seems to be 0.

Namely, I tried a few times to adapt the `malicious-[...].c` files at [`rpl-attacks/templates/experiment/motes`](https://github.com/dhondta/rpl-attacks/tree/master/templates/experiment/motes) to make the malicious mote compile with the option set to multiple test values, but `rpl.log` always shows DIS sent by the malicious mote from the very beginning of the experiment.

I fear this issue comes from Contiki.
